monodbassignment/posts.json

```diff
@@ -0,0 +1,21 @@
+{
+  "id":"String datatype", 
+  "author_id": "String datatype(id of author)",
```

Comment:
Instead of author_id, its a better practice to mention the key name with a reference to its collection. In this case user_id